src/main/kotlin/nl/info/client/brp/PersonenApi.kt

```diff
@@ -57,5 +58,8 @@ interface PersonenApi {
      * Gebruik de fields parameter om alleen die gegevens op te vragen die je nodig hebt en waarvoor je geautoriseerd bent.
      */
     @POST
-    fun personen(personenQuery: PersonenQuery): PersonenQueryResponse
+    fun personen(
+        @HeaderParam(BRPClientHeadersFactory.X_DOELBINDING) purpose: String,
```

Comment:
We might want to swap the order of params. To me the main param is the query and header values (if any) should follow